array_test.go

```diff
@@ -1482,6 +1518,13 @@ func TestArray_Every(t *testing.T) {
 		array.chain.assertNotFailed(t)
 	})
 
+	t.Run("nil func", func(t *testing.T) {
```

Comment:
Let's rename this to "invalid argument" and move to the end of the test, for consistency with other tests.

Reply:
Changed to the consistency of other tests